eman2/wizards.py

```diff
@@ -118,3 +119,19 @@ def show(self, form):
         if myprops['applyChanges'] == 'true':
             for param in params:
                 form.setVar(param, myprops[param + '.value'])
+
+
+class EmanTomoExtractionWizard(EmWizard):
+    _targets = [(EmanProtTomoExtraction, ['boxSize'])]
+
+    def show(self, form):
+        tomoextractProt = form.protocol
```

Comment:
tomoExtractProt?

Reply:
solved in https://github.com/scipion-em/scipion-em-eman2/pull/33/commits/9c158b38a1db38591511411de61b3ca96e084186